testsuite/features/secondary/min_check_patches_install.feature

```diff
@@ -26,19 +26,17 @@ Feature: Display patches
 
   Scenario: Check SLES release 6789 patches
     When I follow the left menu "Patches > Patch List > Relevant"
-    And I follow "andromeda-dummy-6789"
+    And I follow "CL-andromeda-dummy-6789"
```

Comment:
uhmm the clone it might be complicating this stuff. I would consider avoid the clone and create that child channel directly as a primary channel.

Reply:
True, it did give me a bit of a headache with these prefixes